### PR TITLE
Add Mindsensors IR Thermometer sensor

### PR DESCRIPTION
**Pull request by donei34**

### CODE REVIEW

File: sensors/nxt_i2c_sensor_defs.c [hunk 1]
```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		}
 	},
+	[MS_IR_THERMOMETER] = {
+		/**
+		 * @vendor_name: mindsensors.com
+		 * @vendor_part_number: IRThermometer
+		 * @vendor_part_name: IRThermometer
+		 * @vendor_website: http://www.mindsensors.com/products/170-ir-temperature-sensor-for-ev3-or-nxt
+		 * @default_address: 0x2A
```
Comment (dlech) [inline sensors/nxt_i2c_sensor_defs.c]: In linux, the address is shifted to the right one bit compared to RobotC, so the address we need is actually 0x15

Also, 0x15 needs to be added to the list [here](https://github.com/ev3dev/lego-linux-drivers/blob/ev3dev-stretch/sensors/nxt_i2c_sensor_core.c#L488)

File: sensors/nxt_i2c_sensor_defs.c [hunk 2]
```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		}
 	},
+	[MS_IR_THERMOMETER] = {
```
Comment (dlech) [inline sensors/nxt_i2c_sensor_defs.c:2047]: `MS_IR_THERMOMETER` also needs to get added to the lists [here](https://github.com/ev3dev/lego-linux-drivers/blob/ev3dev-stretch/sensors/nxt_i2c_sensor.h#L129) and [here](https://github.com/ev3dev/lego-linux-drivers/blob/ev3dev-stretch/sensors/nxt_i2c_sensor.h#L199)

File: sensors/nxt_i2c_sensor_defs.c [hunk 3]
```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
...
+		/**
+		 * @vendor_name: mindsensors.com
+		 * @vendor_part_number: IRThermometer
+		 * @vendor_part_name: IRThermometer
```
Comment (dlech) [inline sensors/nxt_i2c_sensor_defs.c]: Let's use the same name as on the mindsensors.com website. `IR Temperature Sensor for EV3 or NXT`

File: sensors/nxt_i2c_sensor_defs.c [hunk 4]
```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
...
+		 * @vendor_website: http://www.mindsensors.com/products/170-ir-temperature-sensor-for-ev3-or-nxt
+		 * @default_address: 0x2A
+		 */
+		.name		= MS_IR_THERMOMETER_NAME,
```
Comment (dlech) [inline sensors/nxt_i2c_sensor_defs.c]: `MS_IR_THERMOMETER_NAME` needs to be added to the list [here](https://github.com/ev3dev/lego-linux-drivers/blob/ev3dev-stretch/sensors/nxt_i2c_sensor.h#L162)

**Review by dlech — CHANGES_REQUESTED** [review]
This looks great!

There are just a few non-obvious things that need to be fixed. See inline comments.

**Comment by donei34** [comment]
I believe that all of your points have been fixed, please let me know if there's anything else I can / need to do!
And thanks for all of your help!!

**Comment by dlech** [comment]
I've merged it and we will see what happens.

**Pull request merged**